tests/redirects/base.py

```diff
@@ -46,6 +47,8 @@ def url_test(url, location=None, status_code=requests.codes.moved_permanently,
     :param req_kwargs: Extra arguments to pass to requests.get()
     :param resp_headers: Dict of headers expected in the response.
     :param query: Dict of expected query params in `location` URL.
+    :param allow_redirects: Boolean indicating whether redirects should be followed.
```

Comment:
I would suggest `follow_redirects` instead. `allow_redirects` just seems confusing to me in a test suite mainly for redirects.